Test failure

https://travis-ci.org/vert-x3/vertx-mail-client/jobs/452894885

```
Running io.vertx.ext.mail.MailLocalTest
Starting test: MailLocalTest#mailTestTLSNoTrust 
2018-11-09 15:38:00.354 INFO  [main]org.subethamail.smtp.server.SMTPServer.start:278 - SMTP server *:1587 starting
2018-11-09 15:38:00.358 INFO  [org.subethamail.smtp.server.ServerThread *:1587]org.subethamail.smtp.server.ServerThread.run:67 - SMTP server *:1587 started
2018-11-09 15:38:00.448 WARN  [org.subethamail.smtp.server.Session-/127.0.0.1:35326]org.subethamail.smtp.command.StartTLSCommand.execute:85 - startTLS() failed: javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknown
mail finished 
got exception 
io.vertx.core.impl.NoStackTraceThrowable: connection has been closed by the server

Unhandled exception 
java.lang.AssertionError: didn't get expected exception class io.netty.handler.codec.DecoderException but class io.vertx.core.impl.NoStackTraceThrowable
	at io.vertx.ext.unit.impl.TestContextImpl.reportAssertionError(TestContextImpl.java:362)
	at io.vertx.ext.unit.impl.TestContextImpl.fail(TestContextImpl.java:230)
	at io.vertx.ext.mail.SMTPTestBase.lambda$testException$1(SMTPTestBase.java:204)
	at io.vertx.ext.mail.impl.MailClientImpl.lambda$returnResult$5(MailClientImpl.java:155)
	at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:320)
	at io.vertx.core.impl.EventLoopContext.lambda$executeAsync$0(EventLoopContext.java:38)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:404)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:464)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:884)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
```

--- a/src/test/java/io/vertx/ext/mail/MailLocalTest.java
+++ b/src/test/java/io/vertx/ext/mail/MailLocalTest.java
@@ -19,7 +19,6 @@ package io.vertx.ext.mail;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
-import io.netty.handler.codec.DecoderException;
 import io.vertx.ext.unit.TestContext;
 import io.vertx.ext.unit.junit.VertxUnitRunner;
 
@@ -50,7 +49,7 @@ public class MailLocalTest extends SMTPTestWiser {
   public void mailTestTLSNoTrust(TestContext testContext) {
     this.testContext=testContext;
     MailClient mailClient = MailClient.createNonShared(vertx, configLogin().setStarttls(StartTLSOptions.REQUIRED));
-    testException(mailClient, DecoderException.class);
+    testException(mailClient);
   }
 
   @Test

--- a/src/test/java/io/vertx/ext/mail/MailValidCertWrongHostTest.java
+++ b/src/test/java/io/vertx/ext/mail/MailValidCertWrongHostTest.java
@@ -19,7 +19,6 @@ package io.vertx.ext.mail;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
-import io.netty.handler.codec.DecoderException;
 import io.vertx.ext.unit.TestContext;
 import io.vertx.ext.unit.junit.VertxUnitRunner;
 
@@ -41,7 +40,7 @@ public class MailValidCertWrongHostTest extends SMTPTestDummy {
     final MailConfig config = defaultConfig().setHostname("127.0.0.1").setPort(1587).setStarttls(StartTLSOptions.REQUIRED)
         .setKeyStore("src/test/resources/certs/client.jks").setKeyStorePassword("password");
     MailClient mailClient = MailClient.createNonShared(vertx, config);
-    testException(mailClient, DecoderException.class);
+    testException(mailClient);
   }
 
   @Test

